akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowLogSpec.scala

```diff
@@ -28,7 +28,7 @@ class FlowLogSpec extends AkkaSpec("akka.loglevel = DEBUG") with ScriptedTest {
 
   "A Log" must {
 
-    val supervisorPath = "akka://FlowLogSpec/user/$a"
+    val supervisorPath = "akka://FlowLogSpec/user/StreamSupervisor-0"
```

Comment:
Hmm true... So if I'd use an Actor's context as factory the supervisor would be living in that part of the actor tree. Makes sense.